dotCMS/src/main/java/com/dotmarketing/portlets/variant/model/Variant.java

```diff
@@ -0,0 +1,45 @@
+package com.dotmarketing.portlets.variant.model;
+
+import java.io.Serializable;
+
+/**
+ * Variants represent branches or workspaces for {@link com.dotmarketing.portlets.contentlet.model.Contentlet}.
+ *
+ * In the future it should include also: {@link com.dotmarketing.portlets.templates.model.Template}
+ * and {@link com.dotmarketing.portlets.containers.model.Container}.
+ *
+ * They are a new dimension in addition to languages which means you can have a content version in
+ * a language and in a variation.
+ */
+public class Variant {
```

Comment:
Why not immutable?

Reply:
done https://github.com/dotCMS/core/pull/22839/commits/8aaaf9e8f7d06c71edfd820737a9fcabc12d94ed#diff-8ca6d1c6e473728ac59237d6732ba7ef6c5e1fdf96fbe0131b6c543122501addR1

Reply:
approved